boards/sensoan/sgw2/sgw2_nrf9160_common.dtsi

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ * Copyright (c) 2024 Sensoan Oy
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include "sgw2_nrf9160.h"
+#include "sgw2_common.dtsi"
+#include "sgw2_nrf9160_common-pinctrl.dtsi"
+
+/ {
+	model = "SGW2 nRF9160";
+	compatible = "sensoan,sgw2-nrf9160";
+
+	chosen {
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+		zephyr,uart-mcumgr = &uart0;
+	};
+
+	buttons {
+		compatible = "gpio-keys";
+
+		button0: button_0 {
+			gpios = <&edge_connector 11 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
+			label = "Button 1";
+			zephyr,code = <INPUT_KEY_0>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		led_red: led_0 {
+			gpios = <&edge_connector 7 GPIO_ACTIVE_LOW>;
+			label = "RGB red channel";
+		};
+		led_green: led_1 {
+			gpios = <&edge_connector 8 GPIO_ACTIVE_LOW>;
+			label = "RGB green channel";
+		};
+		led_blue: led_2 {
+			gpios = <&edge_connector 9 GPIO_ACTIVE_LOW>;
+			label = "RGB blue channel";
+		};
+	};
+
+	pwmleds {
+		compatible = "pwm-leds";
+		pwm_led0: pwm_led_0 {
+			pwms = <&pwm0 0 PWM_MSEC(8) (GPIO_ACTIVE_LOW | PWM_POLARITY_NORMAL)>;
+		};
+		pwm_led1: pwm_led_1 {
+			pwms = <&pwm0 1 PWM_MSEC(8) (GPIO_ACTIVE_LOW | PWM_POLARITY_NORMAL)>;
+		};
+		pwm_led2: pwm_led_2 {
+			pwms = <&pwm0 2 PWM_MSEC(8) (GPIO_ACTIVE_LOW | PWM_POLARITY_NORMAL)>;
+		};
+	};
+
+	nrf5340_reset: gpio-reset {
+		compatible = "sensoan,sgw2-nrf5340-reset";
+		status = "okay";
+		gpios = <&gpio0 26 GPIO_ACTIVE_LOW>;
+	};
+
+	edge_connector: edge_connector {
+		compatible = "gpio-nexus";
+		#gpio-cells = <2>;
+		gpio-map-mask = <0xffffffff 0xffffffc0>;
+		gpio-map-pass-thru = <0 0x3f>;
+		gpio-map = <7 0 GPIO_AND_PIN(EDGE_CONN, 7) 0>,	/* LED_Red */
```

Comment:
text is not aligned

Reply:
If you check in the review tab here https://github.com/zephyrproject-rtos/zephyr/pull/81437/files#diff-3b7bdd3eba91830df7d72e7821cef67abb42fccc74502b65af274ef3dc169f06R72 the green and blue options are indented more than the rest so they stick out, not required to fix anyhow

Reply:
text is still not aligned but not a requirement to fix anyhow, just looks odd